packages/dev/create/bin/create.js

```diff
@@ -0,0 +1,144 @@
+// Check package name directory doesn't already exist.
+if (fs.existsSync(packageDir)) {
+  console.error(`Package directory already exists,
+    Remove ${packageName} and try again.`);
+  process.exit(1);
+}
+
+console.log(`Creating a new Blockly\
+  ${chalk.green(packageType)} in ${chalk.green(root)}.\n`);
+
+// Create the package directory.
+fs.mkdirSync(packageDir);
+
+const templateDir = `../templates/${packageType}/`;
+const templateJson = require(path.join(templateDir, 'template.json'));
+
+const packageJson = {
+  name: `@blockly/${packageType}-${packageName}`,
+  version: `0.${new Date().toISOString().slice(0, 10).replace(/-/g, '')}.0`,
+  description: templateJson.description || `A Blockly ${packageType}.`,
+  scripts: templateJson.scripts || {
+    'build': 'blockly-scripts build',
+    'clean': 'blockly-scripts clean',
+    'dist': 'blockly-scripts build prod',
+    'lint': 'blockly-scripts lint',
+    'prepublishOnly': 'npm run clean && npm run prod',
+    'start': 'blockly-scripts start',
+  },
+  main: './dist/index.js',
+  module: './src/index.js',
+  unpkg: './dist/index.js',
+  author: 'Blockly Team',
+  keywords: ['blockly', packageType, packageName],
+  homepage: `https://github.com/google/blockly-samples/tree/master/packages/${packageType}s/${packageName}#readme`,
+  bugs: {
+    url: 'https://github.com/google/blockly-samples/issues',
+  },
+  repository: {
+    'type': 'git',
+    'url': 'https://github.com/google/blockly-samples.git',
+    'directory': `packages/${packageType}s/${packageName}`,
+  },
+  license: 'Apache-2.0',
+  directories: {
+    'dist': 'dist',
+    'src': 'src',
+  },
+  files: [
+    'dist',
+    'src',
+  ],
+  devDependencies: { },
+  peerDependencies: templateJson.peerDependencies || {
+    'blockly': '>=3.20200123.0',
+  },
+  publishConfig: {
+    'access': 'public',
+    'registry': 'https://wombat-dressing-room.appspot.com',
+  },
+  engines: {
+    'node': '>=8.17.0',
+  },
+};
+
+// Add dev dependencies.
+const devDependencies = ['blockly', '@blockly/dev-scripts']
+    .concat(Object.keys(templateJson.devDependencies));
+devDependencies.forEach((dep) => {
+  const latestVersion = execSync(`npm show ${dep} version`).toString();
+  packageJson.devDependencies[dep] = `^${latestVersion.trim()}`;
+});
+
+// Write the package.json to the new package.
+fs.writeFileSync(path.join(packageDir, 'package.json'),
+    JSON.stringify(packageJson, null, 2));
+
+// Write the README.md to the new package.
+let readme = fs.readFileSync(path.resolve(__dirname, templateDir, 'README.md'),
+    'utf-8');
+readme = readme.replace(/template/gmi, packageName);
+fs.writeFileSync(path.join(packageDir, 'README.md'), readme, 'utf-8');
+
+// Copy the rest of the template folder into the new package.
+fs.copySync(path.resolve(__dirname, templateDir, 'template'), packageDir);
+
+// Run npm install.
+execSync(`cd ${packageName} && npm install`, {stdio: [0, 1, 2]});
+
+console.log('Done.');
```

Comment:
Add a closing message that points them at the remaining todos.  I suggest:
"Package created.
Search 'TODO' in the new folder to see remaining tasks."

Reply:
Sounds good.